lib/phlex/sgml.rb

```diff
@@ -113,22 +113,24 @@ def __final_call__(buffer = +"", context: Phlex::Context.new, view_context: nil,
 
 			return "" unless render?
 
-			around_template do
-				if block
-					if is_a?(DeferredRender)
-						__vanish__(self, &block)
-						view_template
-					else
-						view_template do |*args|
-							if args.length > 0
-								yield_content_with_args(*args, &block)
-							else
-								yield_content(&block)
+			@_context.around_render do
```

Comment:
Adding another block here will likely cause significant slow downs. I wanted to use a block to implement the whole selective rendering feature so the element would call something like

```
@context.render_element(attribute[:id]) do
  rendering code here
end
```

And the context would determine whether to yield or not. The performance cost wasn't worth it.

Reply:
Looks like I was wrong about this.